README.org

```diff
@@ -179,7 +179,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
    # ATTENTION IF YOU ARE CONTRIBUTOR: This parent category "Interface
    # Enhancement" is frequently abused. Please DO NOT casually place
    # package in this parent category, unless the package is indeed not
```

Comment:
@eric-lac, can you update the PR further?

This section looks like more grammar changes are needed as well. Can you give a run over and reword this section too?

Something like
```
Please DO NOT casually place package in this parent category, unless the package is indeed not belong to any other subcategory, or other category.
```

to
```
Please DO NOT casually place packages in this parent category, unless the package indeed does not belong to any other subcategory or other category.
```

The package vs packages, singular vs plural, and how the diction flows could be improved. Thanks for contributing improvements here.

Reply:
Use whatever creative license / creativity you want to use here. Thanks!